### DISCUSSION TITLE
Insights with filter based on related field

### DISCUSSION THREAD

**Discussion by DigNZ** [directus/directus#15250] [Archive]
With Insights we can use the global variable panel and then reference that in the other panels, but its just free text. It would be great if this was a dropdown that can take items from a collection.

Here is an example. I have data with a list of cars and how far they have been driving and how much fuel they are using. If I want to use the same dashboard for multiple cars then I need the end user to type in the Rego or some unique identifier then the charts and metrics can update to show that specific car. But it would be great if there was just a dropdown list or an item picker so the car can be picked in a foolproof manner, and maybe multiple cars can be picked or whatever. Also means that its permissions / roles friendly.

**Comment by rijkvanzanten** [comment]
I believe your idea is the same as feature request https://github.com/directus/directus/discussions/14194 🙂

**Reply by DigNZ** [reply]
That is exactly the same idea! Thanks!
